crates/stages/src/stages/headers.rs

```diff
@@ -323,6 +323,15 @@ where
         let highest_block = static_file_provider
             .get_highest_static_file_block(StaticFileSegment::Headers)
             .unwrap_or_default();
+
+        provider.unwind_table_by_walker::<tables::CanonicalHeaders, tables::HeaderNumbers>(
+            input.unwind_to + 1,
+        )?;
```

Comment:
this deletes the entries in `tables::HeaderNumbers`, but we also do that below on line 342

Reply:
now we just use the first call, which should be sufficient

Reply:
it turns out this was required because we may be unwinding past the current entries in the db